Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,290 @@
+import UIKit
+import SnapKit
+import SofaAcademic
+
+
+class EventView: BaseView {
+	private var _match : Event
+	private var homeTeamImage = UIImageView()
```

Comment:
generalno pravilo u swiftu kod nazivanja UI komponenata je da uvijek zavrsavaju imena sa tipom koji su oni, npr tu bi trebalo biti 
```suggestion
	private var homeTeamImageView = UIImageView()
```